app/jobs/send_welcome_email.rb

```diff
@@ -1,5 +1,6 @@
 class SendWelcomeEmail < Struct.new(:person_id, :community_id)
 
+  include SessionContextSerializer
```

Comment:
Why did you want to include the mixin even for jobs that don't make external API calls?